versioned_docs/version-2.x/appendices/migration-guides/index.mdx

```diff
@@ -12,6 +12,20 @@ import ContactLink from "@site/src/components/ContactLink";
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
+## `2.23.0` -> `2.24.0`
+
+### CSP directives
```

Comment:
strictly speaking, while migrating from 2.23.0 to 2.24.0 there's nothing to change in that regard. That said, in previous releases we took the habit to have a "New features" paragraph, I think this is where you could advertise the new CSP directives.